semgrep-core/src/parsing/pfff/java_to_generic.ml

```diff
@@ -425,17 +425,36 @@ and resources (t1, v, t2) =
   |> G.s
 
 and stmt st =
+  match stmt_aux st with
```

Comment:
You could reuse Ast_generic.stmt1?
or it's because you you want your special comment?

Reply:
I do want my special comment :)